modules/SofaConstraint/src/SofaConstraint/GenericConstraintSolver.cpp

```diff
@@ -260,9 +258,8 @@ bool GenericConstraintSolver::buildSystem(const core::ConstraintParams *cParams,
 
     if (unbuilt.getValue())
     {
-        for (unsigned int i=0;i<constraintCorrections.size();i++)
+        for (auto cc : constraintCorrections)
```

Comment:
and from here, I guess it should be auto* cc ? ( but I guess in this case it does not matter actually)